bootstrap.sh

```diff
@@ -24,6 +24,14 @@ echo "node versions: $NODE_VERSIONS"
 echo "nvm version: $NVM_VERSION"
 
 bootstrap() {
+  echo "--- setting system preferences."
+  sudo systemsetup -settimezone GMT
+  sudo systemsetup -setsleep Off
+  sudo systemsetup -setcomputersleep Off
+  sudo systemsetup -setdisplaysleep Off
+  sudo systemsetup -setharddisksleep Off
+  sudo systemsetup -setremotelogin on
```

Comment:
Not sure if it's the right place but could we also disable App Nap i.e.

```defaults write NSGlobalDomain NSAppSleepDisabled -bool YES```

More details about App Nap can be found here: http://osxdaily.com/2014/05/13/disable-app-nap-mac-os-x/

Thank you!